hummingbot/client/command/connect_command.py

```diff
@@ -31,35 +33,38 @@ async def connect_exchange(self,  # type: HummingbotApplication
         self.app.clear_input()
         self.placeholder_mode = True
         self.app.hide_input = True
+        paper_trade = "testnet" if global_config_map.get("paper_trade_enabled").value else None
         if exchange == "kraken":
             self._notify("Reminder: Please ensure your Kraken API Key Nonce Window is at least 10.")
         exchange_configs = [c for c in global_config_map.values()
                             if c.key in settings.CONNECTOR_SETTINGS[exchange].config_keys and c.is_connect_key]
         to_connect = True
         if Security.encrypted_file_exists(exchange_configs[0].key):
             await Security.wait_til_decryption_done()
+
             api_key_config = [c for c in exchange_configs if "api_key" in c.key][0]
             api_key = Security.decrypted_value(api_key_config.key)
-            answer = await self.app.prompt(prompt=f"Would you like to replace your existing {exchange} API key "
+            answer = await self.app.prompt(prompt=f"Would you like to replace your existing {exchange} {paper_trade} API key "
```

Comment:
The problem with this is most our connectors have no `testnet`. Also we don't want to change the meaning of what paper trade is. Paper trade mode is HB gets order book data from exchanges, but maintain trading and balances itself which is different from `testnet`. Actually, users should not be using `paper trade` mode on `testnet` as it will confuse them when they do see their orders on the exchange.

Reply:
Yeah, I also missed this when cleaning up after I added testnet like you did for Binance.